retro/__init__.py

```diff
@@ -8,8 +8,14 @@
 ROOT_DIR = os.path.abspath(os.path.dirname(__file__))
 core_path(os.path.join(os.path.dirname(__file__), "cores"))
 
-with open(os.path.join(os.path.dirname(__file__), "../VERSION")) as f:
-    __version__ = f.read()
+try:
```

Comment:
Given that we have fixed this bug, could we remove the try except and the extra if statement at the end?

Reply:
@pseudo-rnd-thoughts  unfortunately the bug is still present. The reason is because there is no `VERSION` file included in the install at all. I've included the `VERSION` file in `MANIFEST.in` and made the `setup.py` edit you suggested and still no luck. What is odd is that the `Version` field of `pip show stable-retro` is populated with the correct value. Any ideas? Please see the screenshot for reference


<img width="1004" alt="Screenshot 2023-07-03 at 7 07 13 PM" src="https://github.com/Farama-Foundation/stable-retro/assets/49871927/070eaad2-4d18-4d91-aef8-b66bc9941533">